### ISSUE TITLE
Window is blank

### ISSUE THREAD

**Issue by maxi322** [VSCodium/vscodium#1701]
**Describe the bug**
Since the latest version of VSCodium for Debian the window of the application is only displayed blank as shown in the screenshot.
Only when downgrading to version 1.81.0.23216 it works again.

**Please confirm that this problem is VSCodium-specific**
- [x] This bug doesn't happen if I use Microsoft's Visual Studio Code. It only happens in VSCodium.

**Please confirm that the issue/resolution isn't already documented**
- [x] I checked [the Docs page](https://github.com/VSCodium/vscodium/blob/master/DOCS.md) and my issue is not mentioned there.

**To Reproduce**
Update to the latest version.

**Expected behavior**
To work as before.

**Screenshots**
![Bildschirmfoto vom 2023-10-24 23-06-16](https://github.com/VSCodium/vscodium/assets/66129899/3c060a03-ce5f-4e15-9c0b-e9a8e850b43e)

**Desktop (please complete the following information):**
 - OS: Debian sid
 - Architecture amd64
 - Version 1.83.1.23285
 - App Manager apt
 - Sandboxed no

**Comment by daiyam** [comment]
Have you tried `rm -rf .config/VSCodium/GPUCache` from [Troubleshooting](https://github.com/VSCodium/vscodium/wiki/Troubleshooting#graphical-artifacts)?

**Comment by maxi322** [comment]
> Have you tried `rm -rf .config/VSCodium/GPUCache` from [Troubleshooting](https://github.com/VSCodium/vscodium/wiki/Troubleshooting#graphical-artifacts)?

Yes this indeed solves the problem. Thanks as well!
The question is now if the deletion of the GPUCache should become part of for example the postinst script of the debian package. That way this issue could be avoided in the future.

**Comment by APPLESOFT98** [comment]
I had the same issue after I've updated to the latest version, doing `rm -rf .config/VSCodium/GPUCache` seems to have solved the problem for now.

- OS: OpenSUSE Tumbleweed
- Architecture: amd64
- Version: 1.83.1.23285
- App manager: Zypper
- Sandboxed: no

**Comment by Askaniy** [comment]
Same problem, exactly the same configuration and the same solution as in the previous post. I have a Wayland session, if it would help.

**Comment by gdcmarinho** [comment]
@daiyam I think this issue can be closed.

**Comment by dieub56** [comment]
Had the same problem, @nilsnh offered a solution for flapak users in issue #1692
To run the app : `flatpak run com.vscodium.codium --disable-gpu`
"Permanent" fix : `rm -rf ~/.var/app/com.vscodium.codium/config/VSCodium/GPUCache`

**Comment by APPLESOFT98** [comment]
I just updated to 1.84.0.23306 and I'm experiencing the same problem again. 
Unfortunately it seems that `rm -rf .config/VSCodium/GPUCache` is only a temporary solution and this bug does not seem to be fixed permanently.

@daiyam I think this issue should be reopened as it's not fixed.

- OS: OpenSUSE Tumbleweed
- Architecture: amd64
- Version: 1.84.0.23306
- App manager: Zypper
- Sandboxed: no

**Comment by ccoenen** [comment]
A similar problem is apparently introduced by Electron 25 into Element and a few other Electron based programs:

* https://github.com/flathub/im.riot.Riot/issues/411
* https://github.com/microsoft/vscode/issues/190437

**Comment by APPLESOFT98** [comment]
This time I wanted to wait a bit before upgrading VSCodium from version 1.84.0.23306 to 1.84.2.23319, and this time I didn't experience this problem. It seems that the problem is fixed but I don't want to bring bad luck.

- OS: OpenSUSE Tumbleweed
- Architecture: amd64
- Version: 1.84.2.23319
- App manager: Zypper
- Sandboxed: no

**Comment by APPLESOFT98** [comment]
> This time I wanted to wait a bit before upgrading VSCodium from version 1.84.0.23306 to 1.84.2.23319, and this time I didn't experience this problem. It seems that the problem is fixed but I don't want to bring bad luck.
> 
>     * OS: OpenSUSE Tumbleweed
>     * Architecture: amd64
>     * Version: 1.84.2.23319
>     * App manager: Zypper
>     * Sandboxed: no

And I spoke too soon, this time the problem did not arise immediately after the update but after 2 days from the update randomly. 
If we know that the problem is present on Electron 25.9.2, is it possible to get around this by flushing the GPUCache in a post-install/upgrade script each time or, upgrading Codium to a newer version of Electron where the issue has been resolved?

**Comment by APPLESOFT98** [comment]
Although VSCodium hasn't been updated in a while, I'm still experiencing this issue _randomly_ now, instead of every update. 
I'm still at the latest version 1.84.2.23319 and the temporary solution of removing the GPU cache folder still works, but if I have to continue to have this problem either after each update or randomly now, I temporarily install Notepadqq or directly Microsoft VSCode until the problem is permanently resolved on Codium. We can't go on like this...

- OS: OpenSUSE Tumbleweed
- Architecture: x86_64/AMD64
- Version: 1.84.2.23319
- Package manager: Zypper
- Sandboxed: no

**Comment by duke-silver** [comment]
> yep, I just experienced this out of the blue, I thought it was something I did to my system by accident but seems not (thankfully)
> 
> `rm -rf .config/VSCodium/GPUCache` as stated previously works fine for me
> 
> I'm also on tumbleweed, codium version `1.84.2` and commit `1fc52b725b3f76b97eadfd988cea42a739ae923f` it was updated around 2 weeks ago.

Same exactly. GPUCache deletion solved.

**Comment by daiyam** [comment]
https://github.com/microsoft/vscode/issues/190437 has been fixed and merged. The next release should resolve the issue since the Insiders version seems to fix the issue.

**Comment by APPLESOFT98** [comment]
Just updated to 1.85.0.23343 and on the first launch after the update I didn't encounter the problem. Since this problem also occurred **_randomly_** after the update to 1.84.2.23319, and not **_immediately_** after, I recommend waiting a little longer to confirm that the problem has been fixed on Codium. 
I'll try to use Codium more to see if the problem doesn't show up anymore, but just since it didn't show up immediately after the update it's a good sign.


- OS: OpenSUSE Tumbleweed
- Architecture: x86_64/AMD64
- Version: 1.85.0.23343
- App manager: Zypper
- Sandboxed: No

**Comment by APPLESOFT98** [comment]
So, I used Codium more intensively to check if the problem was resolved, and I also updated to 1.85.1.23348 on both my laptop with OpenSUSE Tumbleweed, a second computer still with Tumbleweed, and my Mac Mini running MacOS 10.15.7. Both by trying it with various projects and having updated it manually on each platform, it seems that the issue has **actually** been solved this time. 

@daiyam  I think we can finally close this issue after 2 months, and in case it occurs again I will report it here.
